shared/tx-pool-executor/src/tx_pool_executor.rs

```diff
@@ -100,8 +94,8 @@ impl TxPoolExecutor {
 
         // immediately return if resolved_txs is empty
         if resolved_txs.is_empty() && cached_txs.is_empty() {
-            let (_tx, err) = unresolvable_txs.get(0).expect("unresolved tx exists");
-            return Err(err.to_owned());
+            let (_tx, err) = unresolvable_txs.remove(0);
```

Comment:
the old one is better.

Reply:
`unresolvable_txs.get(0).expect("unresolved tx exists")`

return a reference `&Error`, and clone below.
But the new error type `ckb_error::Error` is not clonable. So I have to use `Vec::remove` to extract the error entity.

Reply:
what about deriving clone trait for `Error`?

Reply:
We have to derive Clone manually.
Updated in commit https://github.com/nervosnetwork/ckb/pull/1344/commits/750b0eb03618c918d2eb4dbdaf710d9f9a5b65a6 , PTAL